features/release.feature

```diff
@@ -367,6 +367,36 @@ MARKDOWN
           ASSET_TARBALL
           """
 
+  Scenario: Delete a release
+    Given the GitHub API server:
+      """
+      get('/repos/mislav/will_paginate/releases') {
+          json [
+            { url: 'https://api.github.com/repos/mislav/will_paginate/releases/123',
+              upload_url: 'https://api.github.com/uploads/assets{?name,label}',
+              tag_name: 'v1.2.0',
+              name: 'will_paginate 1.2.0',
+              draft: true,
+              prerelease: false,
+              assets: [
+                { url: 'https://api.github.com/repos/mislav/will_paginate/assets/9876',
```

Comment:
It's fine that you copied the full payload from GitHub's API, but typically in tests I reduce the JSON payload to just fields we read. In hub we don't actually read most JSON fields, so it's fine if they're not there at all. It makes the tests read cleaner.

For this payload, it would be fine if we only kept the fields `id`, `url`, and `tag_name`.